index.html

```diff
@@ -195,12 +195,44 @@ <h2 class="card-text2">Skills</h2>
 
 
    <!-- get in touch -->
-   <div class="contact">
-     <p class="contact-info">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
 
-   </div>
+   <div class="contact-container">
+     <div class="contact-message">
+       <p class="contact-info">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+     </div>
+     <div class="contact-forms">
+       <form class="form" action="https://formspree.io/f/xrgdnqoz" method="post">
+         <ul class="inputs">
+           <li>
+             <label for="name" class="label"></label>
+             <input type="text" name="user-name" value="" id="name" placeholder="Full Name" maxlength="30" class="in">
+           </li>
+
+           <li>
+            <label for="mail" class="label"></label>
+            <input type="email" name="user-email" value="" id="e-mail" placeholder="E-mail address" class="in">
+           </li>
+
+           <li>
+            <label for="msg" class="label"></label>
+            <textarea name="user-message" rows="" cols="" id="msg" placeholder="Enter text here" maxlength="500" class="in"></textarea>
+          </li>
+         </ul>
```

Comment:
- [ ] Please kindly add `require` attribute to all the input fields to ensure proper validation. 👍